### DISCUSSION TITLE
trader bins show stockpile name

### DISCUSSION THREAD

**Discussion by tily-lass** [DFHack/dfhack#3224] [Ideas]
when sorting through bins to move to the trader, it will have "XYZ BIN (##)" where ## is the stockpile number.  Is there aaaaany ways to get it to be "XYZ BIN (stockpile name)" instead? because I will be damned if i can remember stockpile numbers =/

**Comment by quietust** [comment]
1. Only on the Trade screen, or on **every** screen?
2. Append the stockpile name _after_ the stockpile ID, or **replace** the ID with the name?

Appending the stockpile name and making it apply everywhere would be easiest to implement. Anything more could significantly increase the complexity.
